.circleci/config.yml

```diff
@@ -204,6 +223,7 @@ jobs:
   publish_npm:
```

Comment:
`working_directory` should be updated here as well

Reply:
I'm pointing to the line __above__ the `publish_npm` line - it still has `/var/`